dbms/src/Storages/Page/workload/MainEntry.cpp

```diff
@@ -24,31 +24,32 @@ int StressWorkload::mainEntry(int argc, char ** argv)
         // we do this to trigger REGISTER_WORKLOAD explicitly.
         void _work_load_register_named_HeavyMemoryCostInGC();
         void (*f)() = _work_load_register_named_HeavyMemoryCostInGC;
-        (void)f;
+        f();
         void _work_load_register_named_HeavyRead();
         f = _work_load_register_named_HeavyRead;
-        (void)f;
+        f();
         void _work_load_register_named_HeavySkewWriteRead();
         f = _work_load_register_named_HeavySkewWriteRead;
-        (void)f;
+        f();
+        ;
```

Comment:
duplicate `;`

Reply:
In this way, may be we can just remove `__attribute__((constructor))` in `#define REGISTER_WORKLOAD(WORKLOAD)` ?